bash/scripts/commit_description_parser.sh

```diff
@@ -10,25 +10,47 @@ main() {
 }
 
 # parse-commit-description parses a commit description for any required sibling
-# to pass along to downstream job(s)...
+# commits to pass along to downstream job(s)...
 parse-commit-description() {
   description="${1}"
 
-  # Looks specifically for matches of '[rR]equires <repo>#<sha>',
-  # e.g., "requires builder#abc1234, Requires router#def5678"
-  reqs=$(echo "${description}" | grep -o "[Rr]equires [-a-z]*#[a-z0-9]*" | grep -o "[-a-z]*#[a-z0-9]*") || true
+  # Looks specifically for matches of '[rR]equires <repo>#<prNumber>',
+  # e.g., "requires builder#123, Requires router#567"
+  reqs=$(echo "${description}" | grep -o "[Rr]equires [-a-z]*#[0-9]*" | grep -o "[-a-z]*#[0-9]*") || true
 
-  # split on whitespace into array of '<repo>#<sha>' values
+  # split on whitespace into array of '<repo>#<prNumber>' values
   reqsArray=(${reqs// / })
 
-  # for each '<repo>#<sha>' value in array
+  # for each '<repo>#<prNumber>' value in array
   for i in "${reqsArray[@]}"; do
```

Comment:
Above this you should reset `i`:

```
local i
```

If you don't, `i` can contain something else... and you'd process it below.